### PR TITLE
Fix handling of CTRL+L and url manipulation

### LINKED ISSUE #231: Update jquery.terminal to fix shortcut bug
See https://github.com/jcubic/jquery.terminal/releases/tag/2.17.3 (I am referring to <kbd>CTRL</kbd>+<kbd>L</kbd>)

### PR DESCRIPTION
**Pull request by septatrix**
Fixes #231. The problem seemed to be the old handling of <kbd>CTRL</kbd>+<kbd>L</kbd> and whilst at it I also fixed and cleaned up the url manipulation too

### CODE REVIEW

File: web/index.html [hunk 1]
```diff
@@ -104,14 +107,14 @@
...
+          // clear: false, // do not include 'clear' command
+          // exit: false, // do not include 'exit' command
           checkArity: false,
           historySize: 200,
-          historyFilter: function(line) {
+          historyFilter(line) {
             return line.trim() !== "";
           },
-          completion: function(inp, cb) {
+          completion(inp, cb) {
```
Comment (sharkdp) [inline web/index.html:117]: how... does that even work? :smile: 

What is the advantage over the old `completion: function(…) { … }`? Is this supported in most browsers?

**Comment by sharkdp** [comment]
Thank you very much for your contribution!

This looks (and works) great! I added one inline question

File: web/index.html (hunk 1 above)
Comment (septatrix) [reply web/index.html:117]: It's basically just a shorter version and got added to object literals in ES6. Therefore it is supported every browser except IE. Changing it was primarily just a habit

Comment (sharkdp) [reply web/index.html:117]: Ok - thanks

**Pull request merged**